core/vtk/ttkMergeTreeTemporalReductionEncoding/ttk.module

```diff
@@ -0,0 +1,13 @@
+NAME
+  ttkMergeTreeTemporalReductionEncoding
+SOURCES
+  ttkMergeTreeTemporalReductionEncoding.cpp
+HEADERS
+  ttkMergeTreeTemporalReductionEncoding.h
+  ttkMergeTreeTemporalReductionUtils.h
+DEPENDS
+  mergeTreeTemporalReductionEncoding
+  ttkAlgorithm
+  ttkFTMTree
+  ttkPlanarGraphLayout
+  dimensionReduction
```

Comment:
I'm curious about that dep...
do you use it?

Reply:
(to remove)

Reply:
Removed in the next commit, thanks!